client/README.md

```diff
@@ -66,11 +72,37 @@ export IPFS_API_KEY=<user>:<password>
 ./register-businesses.py
 ```
 
-or locally:
+or locally
 
 ```
 # you may need to run 'ipfs init'
 ipfs daemon
-./bot-communities.py init --ipfs-local
-./register-businesses.py  --ipfs-local
+./bot-communities.py --ipfs-local init 
+./register-businesses.py --ipfs-local
+```
+
+In IPFS, the community icons and data of businesses and offerings are stored.
+
+You can cat/get the data stored in ipfs locally:
+```
+ipfs cat CONTENT_IDENTIFIER
+```
+Or if it was stored remotely (on Infura):
+```
+curl -X POST "https://ipfs.infura.io:5001/api/v0/cat?arg=QmaS7notdEkVmFvL2E19r8bpcVjdgjEV3gu6C2jYqUS1UB" 
+```
+## Notes
+
+The cli provides helpful information for the ordering of options and commands. <br>
+The following options exist:
+* <kbd>--client</kbd> --> (string) path/to/chain/client
+* <kbd>--port</kbd> --> (string) specify port for the chain client
+* <kbd>-l,--ipfs_local</kbd> --> (bool) choose local ipfs node
+* <kbd>--node_url</kbd> --> (string) choose local ipfs node
+
```

Comment:
I am unsure if we should list the options. This kind of information gets outdated easiliy.